vita3k/gui/src/settings_dialog.cpp

```diff
@@ -202,6 +203,7 @@ static CPUBackend set_cpu_backend(std::string &cpu_backend) {
 }
 
 static int current_aniso_filter_log, max_aniso_filter_log;
+static int audio_backend_idx;
```

Comment:
no need news line here ^^
```suggestion
static int audio_backend_idx, current_aniso_filter_log, max_aniso_filter_log;
```